release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1308,6 +1308,10 @@ $ ./openshift-install wait-for install-complete
 
 * Due to an unresolved metadata API issue, you cannot install clusters that use bare-metal workers on {rh-openstack} 16.1. Clusters on {rh-openstack} 16.2 are not impacted by this issue. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2033953[*BZ#2033953*])
 
+* When using the Technology Preview OCI feature with the oc-mirror CLI plug-in, the directory for the OCI catalog is not created properly if you specify a relative directory. As a workaround, specify the full path to the directory for the OCI catalog. (link:https://issues.redhat.com/browse/OCPBUGS-4516[*OCPBUGS-4516*])
```

Comment:
```suggestion
* When using the Technology Preview OCI feature with the `oc-mirror` CLI plug-in, the directory for the OCI catalog is not created properly if you specify a relative directory. As a workaround, specify the full path to the directory for the OCI catalog. (link:https://issues.redhat.com/browse/OCPBUGS-4516[*OCPBUGS-4516*])
```
Just a suggestion/question: should oc-mirror be formatted in backticks since it's a technical name?

Reply:
We didn't backtick it throughout the main docs of this (https://docs.openshift.com/container-platform/4.11/installing/disconnected_install/installing-mirroring-disconnected.html) so to keep it consistent, I'll leave it un-backticked here too